tests/e2e/test_linker_run.py

```diff
@@ -21,26 +24,41 @@
 @pytest.mark.parametrize(
     "pipeline_specification, input_data, computing_environment",
     [
-        # local
+        # slurm
         (
             "pipeline.yaml",
             "input_data.yaml",
-            "environment_local.yaml",
+            "environment_slurm.yaml",
         ),
-        # slurm
+        # local
         (
             "pipeline.yaml",
             "input_data.yaml",
-            "environment_slurm.yaml",
+            "environment_local.yaml",
         ),
     ],
 )
 def test_linker_run(pipeline_specification, input_data, computing_environment, capsys):
-    """e2e tests for 'linker run' command"""
+    """e2e tests for 'linker run' command
+
+    NOTE: We use various print statements in this test because they show up in the
+    Jenkins logs.
+    """
     # Create a temporary directory to store results. We cannot use pytest's tmp_path fixture
-    # because other nodes do not have access to it.
-    with tempfile.TemporaryDirectory(dir="tests/e2e/") as results_dir:
-        results_dir = Path(results_dir)
+    # because other nodes do not have access to it. Also, do not use a context manager
```

Comment:
Do we not have any clean-up we need to do by virtue of not using a context manager?

Reply:
I'm honestly not sure. Not much seems to be growing in `/tmp`. Not sure where else to look

Reply:
To clarify, this _does_ mean that the tmpdirs will persist if the tests fail. We should try to remember to go in manually and clean a few things up every so often.

Reply:
My understanding is that `tempfile.mkdtemp()` and then deleting it manually is effectively the same as the `tempfile.TemporaryDirectory()` context manager